### PR TITLE
DB-9825 add SQL shell command to show clock in prompt

### PR DESCRIPTION
**Pull request by hatyo**
- Add a new SQL shell commands: `prompt clock on` and
  `prompt clock off` which shows a clock in the sqlshell
  prompt in client's local time.

### CODE REVIEW

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java [hunk 1]
```diff
@@ -845,4 +846,8 @@ int getCurrentRowNumber(ResultSet rs)
 	public final Object run() {
 		return  getClass().getResourceAsStream(ProductGenusNames.TOOLS_INFO);
 	}
+
+	public void setPromptClock(boolean showPromptClock) {
```
Comment (arnaud-splice) [inline db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java:850]: Shouldn't that be static? Setting a static field from a nonstatic method is weird.

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java [hunk 2]
```diff
@@ -664,13 +661,17 @@ void newResourceInput(String resourceName) {
 	 * used at each carriage return to show that it is still "live"
 	 * when it is reading multi-line input.
 	 */
-	static void doPrompt(boolean newStatement, LocalizedOutput out, String tag) 
+	 static void doPrompt(boolean newStatement, LocalizedOutput out, String tag)
 	 {
+	 	final SimpleDateFormat sdf = new SimpleDateFormat("yyyy.MM.dd HH:mm:ss");
```
Comment (arnaud-splice) [inline db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java:666]: Please use ISO 8601 delimiters for the date: yyyy-MM-dd

Comment (hatyo) [reply db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java:666]: Good point! thanks, I will fix it.

File: db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java (hunk 1 above)
Comment (hatyo) [reply db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/utilMain.java]: Fixed.

**Pull request merged**